frontend/screens/MainScreen.js

```diff
@@ -1,70 +1,27 @@
-import { View, Text, Image, Dimensions, StyleSheet, Pressable, ScrollView } from 'react-native'
-import React, {useEffect, useState} from 'react'
-import ImagesCarousel from '../components/ImagesCarousel'
+import { View, StyleSheet } from 'react-native'
+import React, { useEffect, useState } from 'react'
 import Button from '../components/Button'
-
-const getRandomNote = async (callback) => {
-    try {
-        const response = await fetch(
-            'http://192.168.1.91:8000/notes/getrandomnote',
-            );
-            const json = await response.json();
-            callback(json);
-        } catch (error) {
-            console.error(error);
-        }
-    };
-
-const getImages = async (callback, id) => {
-    try {
-        const response = await fetch(
-            `http://192.168.1.91:8000/notes/getimages/${id}`,
-            );
-            const json = await response.json();
-            callback(json.images);
-        } catch (error) {
-            console.error(error);
-        }
-    };
+import Note from '../components/Note'
+import useNotificationNote from '../hooks/useNotificationNote';
+import useAPI from '../hooks/useAPI';
 
 const MainScreen = () => {
     const [note, setNote] = useState({})
-    const [images, setImages] = useState([])
+    const { fetchData } = useAPI(`notes/getrandomnote`, setNote, console.log);
+
+    useNotificationNote(setNote)
```

Comment:
this will trigger listener for notification. So if there is notification, we can proccess it (we will display it on the screen)